src/prisma/schema.prisma

```diff
@@ -930,6 +935,7 @@ model TokenGate {
   resourceId       Json
   userRole         String?
   accessTypes      String[]          @default([])
+  type             TokenGateType
```

Comment:
suggestion: add 'lit' as the default, that way you don't have to run a migration. (I think this would fail as-is when it gets deployed, because rows exist without the type?)

Reply:
Done